classes/Form.php

```diff
@@ -915,6 +915,11 @@ public function post()
                     $data = $data[$action];
                 }
 
+                // do not execute action, if deactivated
+                if (Utils::isNegative($data)) {
```

Comment:
May I ask why `Utils::isNegative` instead of === false?

Reply:
If you have an interger parameter 0 will never be passed to the action. I think that using `false` is sufficient here.

Reply:
I am not quite sure, if `=== false` would work. Is it parsed to bool automatically?
Good point with `isNegative`. I did not know it was introduced in 1.7

Reply:
I dont know, just try it out.

Reply:
Looking good now.